src/func.jl

```diff
@@ -171,7 +171,13 @@ function FunctionOperator(op,
                     )
 end
 
+# TODO - update_coefficients(L, u, p, t)
```

Comment:
Why todo? This is here?

Reply:
I implemented `update_coefficients<BANG>`. the todo is for `update_coefficients`.

Reply:
hold on lemme stop being lazy and implement `update_coefficients` as well

Reply:
done